plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/ConfigureGraal.kt

```diff
@@ -165,13 +160,32 @@ internal object ConfigureGraal {
         return file
     }
 
-    //TODO probable reflect.json should be configurable and we should have few preconfigured
-    private fun generateReflect(buildDir: File): File {
-        val reflect = ConfigureGraal::class.java.getResource("/${GraalSettings.DEFAULT_REFLECT_FILE_NAME}").readText()
+    private fun Project.getGraalVmImage(): String {
+        val image = runtime.config.image ?: GraalSettings.GRAAL_VM_DOCKER_IMAGE
```

Comment:
I would better move it to `Runtime` as internal function. like `internal fun RuntimeConfig.getImageOrDefault()`